packages/x-date-pickers-pro/src/DateRangeCalendar/DateRangeCalendar.test.tsx

```diff
@@ -449,7 +449,7 @@ describe('<DateRangeCalendar />', () => {
         }
         const initialValue: [any, any] = [
           adapterToUse.date(new Date(2018, 0, 1)),
-          adapterToUse.date(new Date(2018, 0, 7)),
+          adapterToUse.date(new Date(2018, 0, 2)),
```

Comment:
Was this needed? And if yes, why only for touch (mobile) dragging? 🤔

Reply:
This test is weird
In a real browser, when we have the `touchStart` event, shouldn't it be on the element we try to drag ?
In this test, we click on `1` but the 1st target is `2`.

So I changed to try to make it pass but I feel like it's either

-  `executeDateTouchDragWithoutEnd` that should execute  `touchStart` with the coordinates of`target` as the touch target (which is what we do in every other test)
- `handTouchStart` that should use `event.target` instead of `resolveElementFromTouch`

Reply:
> In a real browser, when we have the `touchStart` event, shouldn't it be on the element we try to drag ? In this test, we click on `1` but the 1st target is `2`.

Yes—it should and it is, the element we start the drag on is selected with `screen.getByRole('gridcell', { name: '1' })`. Touch events are always fired on the same/origin target, just with different coordinates, which co-respond to specific dates.

Reply:
OK so do you agree with the fix I did on the test ?

Reply:
I still don't think it was needed 🤔 
Essentially—it doesn't change anything.
We still start the drag on Jan 1st and assert that the validation has successfully been applied to the flipped day when we touch drag over Jan 9th.

Reply:
Sorry, we ended up in a misunderstanding because of mis-alignment of what the original question was on and what change you have landed on.
After looking at the current change—yes, I agree with it 100%. It was my mistake in the first place. 🙈

Reply:
I misunderstood my own response when reading it this morning :laughing: 
OK so we can update the text :+1: